Earthfile

```diff
@@ -183,8 +183,14 @@ generate-code:
     SAVE ARTIFACT ./src/checkerEntrypoints.ml /
     SAVE ARTIFACT ./src/tok.ml AS LOCAL src/tok.ml
     SAVE ARTIFACT ./src/tok.ml /
+    SAVE ARTIFACT ./src/kit.ml AS LOCAL src/kit.ml
+    SAVE ARTIFACT ./src/kit.ml /
+    SAVE ARTIFACT ./src/lqt.ml AS LOCAL src/lqt.ml
+    SAVE ARTIFACT ./src/lqt.ml /
     SAVE ARTIFACT ./src/constants.ml AS LOCAL src/constants.ml
     SAVE ARTIFACT ./src/constants.ml /
+    SAVE ARTIFACT ./src/burrowOrigination.ml AS LOCAL src/burrowOrigination.ml
+    SAVE ARTIFACT ./src/burrowOrigination.ml /
```

Comment:
Ah, yes, sorry, I forgot to change the `Earthfile` when moving the burrow origination into a separate module, thanks for addressing this :+1: I missed it also when I separated the drift derivative calculation (79de464c51c1b8855a6456e546915a6119544954); I have to patch that too :frowning_face: